sources/zsh/history.d/netdiscover

```diff
@@ -0,0 +1,2 @@
+netdiscover -p
+netdiscover -i "$INTERFACE" -r 192.168.1.0/24
```

Comment:
There is a blank line missing at the end of the file :)

Reply:
Shouldn't be an issue, the parser should handle it

Reply:
still added it :)